src/Worker.cpp

```diff
@@ -229,5 +268,4 @@ int Worker::get_bmp_data_unread_len() {
 
 uint8_t *Worker::get_unread_buffer() {
     return bmp_data_buffer + bmp_data_read_len;
-}
-
+}
```

Comment:
Newlines at end of file are kind for cli users!

Reply:
Fixed.